framework/transport/ieee1905_transport/ieee1905_transport_broker.cpp

```diff
@@ -282,24 +280,15 @@ bool BrokerServer::socket_connected(std::shared_ptr<SocketServer> sd)
         return false;
     }
 
-    if (!sd->getUdsPath().empty()) {
-        LOG(DEBUG) << "New connection on " << sd->getUdsPath()
-                   << " fd = " << uintptr_t(new_socket->getSocketFd());
-    } else {
-        LOG(DEBUG) << "New connection from ip = " << new_socket->getPeerIP()
-                   << " port = " << new_socket->getPeerPort()
-                   << " fd = " << uint64_t(new_socket->getSocketFd());
-    }
+    LOG(DEBUG) << "Accepted new connection, fd = " << new_socket->getSocketFd();
 
     // Socket event handlers
     SocketEventLoop::EventHandlers socket_handlers{
         // Handle incoming data
         .on_read =
             [&](SocketEventLoop::EventType socket, SocketEventLoop::EventLoopType &loop) {
-                if (!handle_msg(socket)) {
-                    // NOTE: Do NOT stop the broker on parsing errors...
-                }
-
+                // NOTE: Do NOT stop the broker on parsing errors...
+                handle_msg(socket);
```

Comment:
Why has `if (!handle_msg(socket))` statement been removed? It would be nice to have some debug output.

Reply:
We don't actually want to do anything on failures in this case.
It's safe to assume that `handle_msg` will print the actual error..